canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -750,6 +751,21 @@ export default class PropertiesController {
 		this.propertiesStore.clearSelectedRows(propertyId);
 	}
 
+	/**
+	 * Enable/disable table row move buttons
+	 * enableTableRowMoveButtons() doesn't enable all the buttons. Buttons will be enabled based on row selection
+	 * @param propertyId Id of the property for the control to operate upon
+	 *
+	 */
+	disableTableRowMoveButtons(inPropertyId) {
+		const propertyId = this.convertPropertyId(inPropertyId);
+		this.disableMoveRowButtons[propertyId.name] = true;
```

Comment:
This needs to be stored using propertyId so embedded controls can disable/enable the buttons.  This also needs to be part of redux so we can listen to changes and rerender the control on changes.